iis-wf/iis-wf-export-actionmanager/src/test/java/eu/dnetlib/iis/wf/export/actionmanager/entity/EntityExportWorkflowTest.java

```diff
@@ -0,0 +1,25 @@
+package eu.dnetlib.iis.wf.export.actionmanager.entity;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import eu.dnetlib.iis.common.AbstractOozieWorkflowTestCase;
+import eu.dnetlib.iis.common.IntegrationTest;
+
+/**
+ * @author mhorst
+ */
+@Category(IntegrationTest.class)
+public class EntityExportWorkflowTest extends AbstractOozieWorkflowTestCase {
+
+    @Test
+    public void testExportDatasetEntity() throws Exception {
+        testWorkflow("eu/dnetlib/iis/wf/export/actionmanager/entity/dataset/default");
```

Comment:
Convention I guess. There were several test workflows already defined with `default` name suggesting there is only one default integration test without any specific test cases that could be expressed in subfolder name.

This seemed to be better than `sampledataproducer` used as an alternative convention which is a little bit to wordy.